src/norms.jl

```diff
@@ -71,6 +71,23 @@ _build_norm(Lp, terms::Vector{GenericAffExpr}) = _build_norm(Lp, [terms...])
 # Alias for AffExprs. Short-hand used in operator overloads, etc.
 typealias Norm{P} GenericNorm{P,Float64,Variable}
 
+function getValue{P,C,V}(n::GenericNorm{P,C,V})
+    term_vals = getValue(n.terms)
+    if P == 2
+        return sqrt(sumabs2(term_vals))
+    elseif P == 1
```

Comment:
Is there a reason to carry around all of this untested code below for ``P != 2``?